component/core/src/test/java/org/exoplatform/social/core/utils/MentionUtilsTest.java

```diff
@@ -33,15 +33,15 @@ public void testSubstituteUsernames(){
         String portalOwner = "dw";
         String message = "hello <p>@root</p>";
 
-        message = MentionUtils.substituteUsernames(portalOwner, message);
+        message = MentionUtils.substituteUsernames(portalOwner, message, null);
         assertEquals("hello <p><a href=\"http://test.com/portal/dw/profile/root\" target=\"_parent\">Root Root</a></p>", message);
     }
 
     @Test
     public void testWrongFormatMentionWithSubstituteUsernames(){
         String portalOwner = "dw";
         String message = "hello @ jhon";
-        message = MentionUtils.substituteUsernames(portalOwner, message);
+        message = MentionUtils.substituteUsernames(portalOwner, message, null);
```

Comment:
Delete using null (to apply for all methods please)

```suggestion
        message = MentionUtils.substituteUsernames(portalOwner, message);
```